pkg/fanal/secret/builtin-rules.go

```diff
@@ -104,6 +105,15 @@ var builtinRules = []Rule{
 		SecretGroupName: "secret",
 		Keywords:        []string{"account"},
 	},
+	{
+		ID:              "azure-client-secret",
+		Category:        CategoryAzure,
+		Severity:        "CRITICAL",
+		Title:           "Azure Client Secret",
+		Regex:           MustCompile(`(?P<secret>[a-zA-Z0-9-_.~]{5}~[a-zA-Z0-9-_.~]{34})`),
+		SecretGroupName: "secret",
+		Keywords:        []string{"client_secret", "ARM_CLIENT_SECRET"},
```

Comment:
For example, it will not detect the password in the following command since it doesn't include `client_secret` and `ARM_CLIENT_SECRET` either. Is it OK?
```
az acr import \
  --name myregistry \
  --source sourceregistry.azurecr.io/sourcerrepo:tag \
  --image targetimage:tag \
  --username c23785b0-d426-4b93-b63f-6aa58eb27610 \
  --password WTf8Q~wgIeaP~LbkKH7edueENq2fmXxrCpBV8bMA
```